### ISSUE TITLE
Add keyboard shortcuts/mnemonics/access keys to context menu

### ISSUE THREAD

**Issue by camilstaps** [handsontable/handsontable#6727]
### Description
This is a feature request, not a bug.

In many applications when opening the shortcut menu there are keyboard shortcut available, like <kbd>P</kbd> for "Save Page As..." below. (I believe these are called mnemonics in Gtk and access keys in Windows.) It would be useful to have this in the custom shortcut menu that Handsontable provides as well.

![context menu example](https://user-images.githubusercontent.com/1202754/75166713-96e12c80-5724-11ea-817f-a2606b90f03d.png)

I can see two ways to implement this:

1. Allow a new field of each context menu item record, `mnemonic`, which can contain a character that serves as the mnemonic. Either the user is responsible for formatting the HTML in the name, or Handsontable can do this intelligently by underlining the first matching character in the name.
2. Parse the name and use a special character (like `_` in Gtk) to indicate the mnemonic, so in the above example you'd use `Save _Page As...`.

Option 2 gives shorter code, but is not fully backwards compatible and you would have to implement escaping as well. Option 1 is fully backwards compatible but leads to larger code.

I'm happy to implement either solution, or something else, but I thought it'd be good to first get agreement on whether this is a useful addition to the library and what the API should look like.

Thanks for your consideration!

### Steps to reproduce
1. Create Handsontable with `contextMenu: true` or custom options
2. Right click on cell

### Demo
Example from the [demo](https://handsontable.com/docs/7.4.2/demo-context-menu.html): https://jsfiddle.net/z9vw27bj/

### Your environment
* Handsontable version: 7.4.2
* Browser Name and version: not relevant
* Operating System: not relevant

**Comment by AMBudnik** [comment]
Thank you for sharing this idea. That is a good idea. It will definitely help our users to move faster in the larger menus.

**Comment by camilstaps** [comment]
@AMBudnik thanks! I don't know what your decision process looks like, but if we get an authoritative decision on what the API should look like I could have a look if I can get it implemented, if you want.

**Comment by AMBudnik** [comment]
> I don't know what your decision process looks like

Usually, we plan milestones a **month or two upfront.** 

**Once per year, we are making more ambitious changes**.
Last year we started to work on a side project - a new formula engine that in the upcoming weeks will replace the current formula plugin. At the same time, we added an index mapper that became a center of the index logic and we have made some steps to alter the way cell metadata is being used and processed. As that should take us some time I believe that we will have a bit more time in Q2/2020.

When it comes to the **importance** of tasks. If there are developers available we start with [regression bugs](https://github.com/handsontable/handsontable/issues?q=is%3Aopen+is%3Aissue+label%3ARegression). However this doesn't mean that we won't be able to add a new feature or API as it all depends on the already started process.

In 2019 we decided to start planning changes that will allow users with disabilities to have an easier job with Handsontable. We plan to focus on it in 2020. I belive that the feature that you proposed would be a part of this alteration for the easier access, and it so we probably create an epic that gathers all similar subjects.
